pilot/pkg/security/authz/matcher/header_test.go

```diff
@@ -52,14 +55,55 @@ func TestHeaderMatcher(t *testing.T) {
 				},
 			},
 		},
+		{
+			Name:        "regex match - disabled - ExactMatch",
```

Comment:
just change the name to `regex match disabled`, and the other one to `regex match enabled`